pages/Screenshots.html

```diff
@@ -25,9 +25,9 @@
     <a id="gallery" class="anchor" href="#gallery" aria-hidden="true">
       <span class="octicon octicon-link"></span>
     </a>
-    Gallery
+    Library
   </h3>
-  <p><img src="static/images/gallery.png" alt="Screenshot of gallery page" class="screenshot-image"></p>
+  <p><img src="static/images/library.png" alt="Screenshot of gallery page" class="screenshot-image"></p>
```

Comment:
Yes I will delete the old image but not right now, it is in use in at least one other location so I will only delete it once I have updated the other page that contains this image.